.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ references:
   images:
     # When updating the Go version, remember to also update the versions in the
     # workflows section for go-test-lib jobs.
-    go: &GOLANG_IMAGE docker.mirror.hashicorp.services/circleci/golang:1.17.2
+    go: &GOLANG_IMAGE docker.mirror.hashicorp.services/cimg/go:1.17.5
```

Comment:
New version of CircleCI images: [link](https://circleci.com/developer/images/image/cimg/go)